kaldb/src/main/resources/log4j2.xml

```diff
@@ -4,7 +4,9 @@
 
     <Appenders>
         <Console name="console" target="SYSTEM_OUT">
-            <PatternLayout  pattern="[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %notEmpty{[trace=%X{traceId}/%X{spanId}]} %c{1} - %msg%n" />
+            <!-- https://github.com/apache/logging-log4j2/blob/master/log4j-layout-template-json/src/main/resources/EcsLayout.json -->
+            <!-- MDC keys will get their own field as per EcsLayout.json  -->
+            <JsonTemplateLayout eventTemplateUri="classpath:EcsLayout.json" />
```

Comment:
How hard would it be to emit log messages in span json format and use that as a template?

Reply:
What's a span JSON format? Also we have to then see if that format can be ingested by our internal logging pipeline.